src/modules/wizeng/WizengModule.v3

```diff
@@ -150,6 +169,36 @@ class WizengModule extends HostModule("wizeng") {
 		System.write(1, range.result);
 		return HostResult.Value0;
 	}
+	// Exported: output a range of bytes directly to stdout.
+	def puts_array(args: Range<Value>) -> HostResult {
+		match (args[0]) {
```

Comment:
Since we're validating the function's signature, you can get by with a simpler cast: `HeapArray.!(Value.Ref.!(args[0]).val)` should do.

Reply:
Fixed.